internal/services/redis/redis_cache_resource_test.go

```diff
@@ -1492,7 +1492,7 @@ resource "azurerm_redis_cache" "test" {
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger)
 }
 
-func testCheckSSLInConnectionString(resourceName string, propertyName string, requireSSL bool) acceptance.TestCheckFunc {
+func testCheckSSLInConnectionString(resourceName string, propertyName string) acceptance.TestCheckFunc {
```

Comment:
given that the connection string is now specifically set to be true now, we probably don't need to have this check in the tests at all anymore?

Reply:
Good suggestion, fixed. Thanks.